### PR TITLE
Remove Running Code from Init File

### LINKED ISSUE #17: Can't import / use hovor
Seems as though [this line](https://github.com/dialogue-planning/contingent-plan-executor/blob/184b8c4addc22c6b39e7904813f9c031f5d3d656/contingent_plan_executor/hovor/__init__.py#L27) gets triggered when we're just trying to import hovor stuff (for the simulation things w/ @jlaframboise ).

Any chance you could detail what's going on here, @beckydvn ?

### PR DESCRIPTION
**Pull request by beckydvn**
- Also shuffled some things around to prevent circular imports
- Closes #17

### CODE REVIEW

File: contingent_plan_executor/hovor/__init__.py [hunk 1]
```diff
@@ -20,29 +20,28 @@ def setup(sqlalchemy_db_uri):
...
+    if len(sys.argv) > 1:
+        output_files_path = sys.argv[1]
+    else:
+        raise ValueError("Please provide the directory to your plan4dial output files as a system argument.")
+        # output_files_path = "local_data/updated_gold_standard_bot" # ONLY for local debugging
 
-# save the output files path name
-with open("out_path.txt", "w") as f:
-    f.write(output_files_path)
+    # save the output files path name
```
Comment (haz) [inline contingent_plan_executor/hovor/__init__.py:35]: This a mix of tabs/spaces?

File: contingent_plan_executor/local_run_utils.py [hunk 2]
```diff
@@ -19,7 +19,6 @@ def run_rasa_model_server(output_files_path):
         requests.post("http://localhost:5005/model/parse", json={"text": ""})
     except requests.exceptions.ConnectionError:
         subprocess.Popen(f"rasa run --enable-api -m {output_files_path}/nlu_model.tar.gz", shell=True)
-        sleep(3.0)
```
Comment (haz) [inline contingent_plan_executor/local_run_utils.py:22]: Hrmz...what was this here for before?

**Review by haz — CHANGES_REQUESTED** [review]
Just a couple of small questions...

File: contingent_plan_executor/hovor/__init__.py (hunk 1 above)
Comment (beckydvn) [reply contingent_plan_executor/hovor/__init__.py:35]: I just tabbed everything once (and I just went in to check to make sure the spaces are even). Not sure why it's showing up funky :P

File: contingent_plan_executor/local_run_utils.py (hunk 2 above)
Comment (beckydvn) [reply contingent_plan_executor/local_run_utils.py:22]: On my home PC it always took a few seconds for the rasa model to set up so I just added that so it wouldn't try to run things a million more times than needed when I was testing things and further slow things down. Just a silly hack I forgot to remove (promise that's the only one 😋)

File: contingent_plan_executor/hovor/__init__.py (hunk 1 above)
Comment (haz) [reply contingent_plan_executor/hovor/__init__.py:35]: But when you select those lines is it using spaces? Sometimes tabs can creep in

Comment (beckydvn) [reply contingent_plan_executor/hovor/__init__.py:35]: Don't think I see a difference in my editor - I went and re-tabbed everything in my IDE and it didn't show any file changes. But I just ran black on it, hope that helps!

**Review by haz — APPROVED** [review]

**Pull request merged**